docker/autoscaler/Dockerfile

```diff
@@ -13,14 +11,10 @@ RUN apt-get update \
     && apt-get clean 
 
 # We have to uninstall wrapt this way for Tensorflow compatibility
-COPY ../../python/requirements.txt ./
-COPY ../../python/requirements_autoscaler.txt ./
-COPY $WHEEL_PATH $WHEEL_NAME
+COPY requirements.txt ./
```

Comment:
This is the right idea, and fine for the moment, but if you want to separate the autoscaler further, you might consider autoscaler owning its own `requirements.txt` what can take care of itself (and minimize the other (dev) dependencies which may not be needed by the autoscaler) -- I'd recommend `python/ray/autoscaler/requirements.txt` and deleting `requirements_autoscaler.txt` as a follow-up

Reply:
Alternatively, it may be helpful to think about a _user_ of the autoscaler images, as opposed to someone who is actively _developing_ the autoscaler code.

`python/requirements_autoscaler.txt` should target the developer, `python/ray/autoscaler/requirements.txt` should target the user. If they're the same dependencies, pick whichever you like

Either way, the requirements installed here should be from a user perspective